pulpcore/app/models/status.py

```diff
@@ -11,35 +11,55 @@
 from pulpcore.app.models import BaseModel
 
 
-class ContentAppStatusManager(models.Manager):
+class AppStatusManager(models.Manager):
     def online(self):
         """
-        Returns a queryset of ``ContentAppStatus`` objects that are online.
+        Returns a queryset of objects that are online.
 
-        To be considered 'online', a ContentAppStatus must have a heartbeat timestamp within
+        To be considered 'online', a AppStatus must have a heartbeat timestamp within
         ``settings.CONTENT_APP_TTL`` from now.
 
         Returns:
-            :class:`django.db.models.query.QuerySet`:  A query set of the ``ContentAppStatus``
+            :class:`django.db.models.query.QuerySet`:  A query set of the
                 objects which are considered 'online'.
         """
-        now = timezone.now()
-        age_threshold = now - timedelta(seconds=settings.CONTENT_APP_TTL)
-
+        age_threshold = timezone.now() - self.model.APP_TTL
         return self.filter(last_heartbeat__gte=age_threshold)
 
+    def missing(self, age=None):
+        """
+        Returns a queryset of workers meeting the criteria to be considered 'missing'
+
+        To be considered missing, a worker must have a stale timestamp.  By default, stale is
+        defined here as longer than the ``settings.WORKER_TTL``, or you can specify age as a
```

Comment:
```suggestion
        defined here as longer than the ``self.model.APP_TTL``, or you can specify age as a
```
Might want to consider rewriting this doc-string to change 'worker' to 'object' since this method now refers to the content app, pulpcore-api & task workers.

Reply:
We call them api workers and content workers... And gunicorn talks a lot about workers too. I think we are probably fine...